src/python/WMCore/BossAir/Plugins/PyCondorPlugin.py

```diff
@@ -875,7 +875,7 @@ def makeSubmit(self, jobList):
             jdl.append("+JobLeaseDuration = isUndefined(MachineAttrMaxHibernateTime0) ? 1200 : MachineAttrMaxHibernateTime0\n")
 
             ### print all the variables needed for us to rely on condor userlog
-            jdl.append("job_ad_information_attrs = JobStatus,QDate,EnteredCurrentStatus,JobStartDate,DESIRED_Sites,ExtDESIRED_Sites,WMAgent_JobID,MATCH_EXP_JOBGLIDEIN_CMSSite\n")
+            jdl.append("job_ad_information_attrs = JobStatus,QDate,EnteredCurrentStatus,DESIRED_Sites,ExtDESIRED_Sites,WMAgent_JobID,MATCH_EXP_JOBGLIDEIN_CMSSite\n")
```

Comment:
Very confusing indeed. Just to make sure I understood you.

No matter what we set to `job_ad_information_attrs`, either `MATCH_EXP_JOBGLIDEIN_CMSSite` or `JOBGLIDEIN_CMSSite` would map to the same `JOBGLIDEIN_CMSSite` in the condor log (string), for running jobs. Is that correct?

Hmm, having a look at a couple of logs from production - for running jobs - one of them has the MATCH_blah classAd and the other doesn't.